src/status_im/components/icons/vector_icons.cljs

```diff
@@ -64,7 +64,7 @@
             :icons/dropdown-up         (slurp-svg "./resources/icons/dropdown_up.svg")
             :icons/dropdown            (slurp-svg "./resources/icons/dropdown.svg")
             :icons/grab                (slurp-svg "./resources/icons/grab.svg")
-            :icons/share               (slurp-svg "./resources/icons/share.svg")})
```

Comment:
What's share.svg? Can we remove this file?

Reply:
i think it happened during the conflict, we should keep share and network icons

Reply:
Don't think so, that's probably result of wrong conflict resolving, will fix